go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/kubernetes-sigs/aws-fsx-csi-driver
 
 require (
-	github.com/aws/aws-sdk-go v1.35.7
+	github.com/aws/aws-k8s-tester/e2e/tester v0.0.0-20210607224202-298c17a2707a // indirect
```

Comment:
we shouldn't need this anymore, can you try running go mod tidy

Reply:
Done. Thanks for catching that!